stock_move_line_auto_fill/README.rst

```diff
@@ -0,0 +1,143 @@
+=========================
+Stock Move Line Auto Fill
+=========================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fstock--logistics--workflow-lightgray.png?logo=github
+    :target: https://github.com/OCA/stock-logistics-workflow/tree/12.0/stock_move_line_auto_fill
+    :alt: OCA/stock-logistics-workflow
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/stock-logistics-workflow-12-0/stock-logistics-workflow-12-0-stock_move_line_auto_fill
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/154/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module alow auto fill quantities in picking operations and autoassignment
+lots quantities.
+
+In Odoo, if you schedule to transfer 50 product quantities and only receive 49
```

Comment:
I don't quite understand this statement. It means receive (or transfer) 50 lines of product or 1 product with 50 units, etc.
@jarroyomorales I understand that the readme is interited from older version. But I do find a hard time trying to understand the readme, otherwise have to dig into the code to understand. Is it possible to enhance the readme ? :)

Reply:
It means 50 lines of product, I don't know if the small change I did helps understand it better. By saying just 50 products instead of 50 product quantities I think its easier to understand it means lines.